Firestore/core/src/firebase/firestore/api/settings.h

```diff
@@ -32,6 +32,8 @@ namespace api {
  */
 class Settings {
  public:
+  // Note: a constexpr array of char (`char[]`) doesn't work with older versions
```

Comment:
You might want to make this more specific:

```
// ... doesn't work with Visual Studio 2015.
```

This makes it easier to know what can be unfixed when dropping support for older platforms

Reply:
Good point, done.